recipes/pyopengl/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: pyopengl
+  version: {{ version }}
+
+source:
+  fn: PyOpenGL-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/P/PyOpenGL/PyOpenGL-{{ version }}.tar.gz
+  md5: 0de021941018d46d91e5a8c11c071693
+
+build:
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - OpenGL.GL
+
+about:
+  home: http://pyopengl.sourceforge.net
+  license: BSD-like
+  summary: Standard OpenGL bindings for Python
+
+extra:
+  recipe-maintainers:
+    - almarklein
+    - mcfletch
```

Comment:
We have not chatted about this :) I added Mike because he's the pyopengl dev, so it seemed appropriate. I am fine with doing the actual maintenance of the conda recipe.